### ISSUE TITLE
Replace Microsoft.JSInterop.Json with JsonSerializer

### ISSUE THREAD

**Issue by pranavkm** [aspnet/AspNetCore#9945]
With the few tests I tried, barring some nasty bugs, it looked like System.Text.Json generally works well with server and client size blazor for serializing and deserializing vanilla JSON payloads. We should remove the `Json` type and recommend users migrate to using `System.Text.Json.JsonSerializer`.

**Comment by conficient** [comment]
If you do, please make this a pluggable option so we can use a different library, e.g. Newtonsoft

**Comment by pranavkm** [comment]
@conficient the suggestion is to not have a helper at all. You should be free to do what works best for you.

**Comment by LaughingJohn** [comment]
Any idea how I reference System.Text.Json in a Blazor app? (I'm running preview VS and .net core 3 preview 5. I've had issues with both Microsoft.JSInterop.Json (can't deserialize a keyed collection although it can serialize one) and NewtonSoft (System.Runtime not found).

On a side note once I added Newtonsoft I couldn't get rid of it by just removing the package, it seemed to be hanging around in the bin/obj folders or something - I deleted them and eventually it complained about the namespace.
